app/src/standard/java/com/twilio/voice/quickstart/VoiceActivity.java

```diff
@@ -453,6 +457,13 @@ private View.OnClickListener holdActionFabClickListener() {
     private View.OnClickListener muteActionFabClickListener() {
         return v -> mute();
     }
+    private View.OnClickListener dialpadFabClickListener() {
+        return v -> {
+            Intent intent = new Intent(v.getContext(), DialpadActivity.class);
```

Comment:
This could be simplified by sending it like this..
```
    Intent intent = new Intent(this, TargetActivity.class);
    intent.putExtra("activeCallId", activeCallId);
    startActivity(intent);
 ```